src/api/app/models/bs_request_action.rb

```diff
@@ -866,6 +866,16 @@ def event_parameters
     params
   end
 
+  def build_results
+    return unless source_project
+
+    source_project_object = Project.find_by_name(source_project)
+    source_package_object = Package.find_by_project_and_name(source_project_object.name, source_package)
```

Comment:
Again, a `Project` with the name that is in `source_project` might note exist anymore or be not readable by `User.session` in which case this whole thing will barf.

Reply:
This is fixed, now